[Thu Dec 22 16:57:22.007594 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247] mod_wsgi (pid=27343): Target WSGI script '/var/www/api/wsgi/flask.wsgi' cannot be loaded as Python module.
[Thu Dec 22 16:57:22.007702 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247] mod_wsgi (pid=27343): Exception occurred processing WSGI script '/var/www/api/wsgi/flask.wsgi'.
[Thu Dec 22 16:57:22.007734 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247] Traceback (most recent call last):
[Thu Dec 22 16:57:22.007759 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]   File "/var/www/api/wsgi/flask.wsgi", line 7, in <module>
[Thu Dec 22 16:57:22.007827 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]     from app import app as application
[Thu Dec 22 16:57:22.007837 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]   File "/var/www/api/app.py", line 7, in <module>
[Thu Dec 22 16:57:22.007868 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]     from upload import *
[Thu Dec 22 16:57:22.007876 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]   File "/var/www/api/upload/__init__.py", line 1, in <module>
[Thu Dec 22 16:57:22.007898 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]     from imageUpload import *
[Thu Dec 22 16:57:22.007906 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]   File "/var/www/api/upload/imageUpload.py", line 8, in <module>
[Thu Dec 22 16:57:22.007939 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247]     import createPath
[Thu Dec 22 16:57:22.007959 2016] [:error] [pid 27343:tid 140381700032256] [client 106.222.55.129:10247] ImportError: No module named createPath
[Fri Dec 23 22:38:15.519718 2016] [core:error] [pid 29365:tid 140269705934592] [client 31.192.106.208:56126] AH00126: Invalid URI in request GET <title>phpMyAdmin HTTP/1.1
[Sun Dec 25 15:29:50.774249 2016] [core:error] [pid 6656:tid 140269798254336] [client 169.229.3.91:58394] AH00135: Invalid method in request Ek\xd6\x07L\x95<\x9e\x19b6\xbb\xba\x94)\xf8\x06\xa8\xbaz\xbf\x19zweT\x89\xc3\xbdB\x8a\x12\xd6\xac\x0f\x8aA\xd1>\x06\xbd:'\xa6\xbb<
[Thu Jan 19 15:14:54.087411 2017] [core:error] [pid 20251:tid 140085617993472] [client 118.160.127.165:8677] AH00135: Invalid method in request \xc2\x8b\xa8\xef\x97\x84\xd0k\xd1&=5o\x913\\\x03\x84\xbc\x02\xe0*\x80`|\x07\x18x%\xbe\xe0\xea
[Sat Jan 21 23:02:35.522620 2017] [core:error] [pid 20251:tid 140085508888320] [client 222.141.64.65:47774] AH00126: Invalid URI in request GET HTTP/1.1 HTTP/1.1
[Wed Jan 25 07:56:50.014335 2017] [core:error] [pid 9649:tid 140085483710208] [client 46.105.123.22:38957] AH00126: Invalid URI in request GET HTTP/1.1 HTTP/1.1
[Fri Feb 03 17:47:34.790744 2017] [core:error] [pid 20812:tid 140085617993472] [client 223.255.145.158:44945] AH00126: Invalid URI in request GET HTTP/1.1 HTTP/1.1
[Tue Feb 28 03:52:05.852862 2017] [core:error] [pid 29152:tid 140085659956992] [client 160.16.54.144:53451] AH00135: Invalid method in request AKPQ / HTTP/1.1
[Tue Mar 07 02:21:56.264347 2017] [core:error] [pid 28240:tid 140085584422656] [client 223.255.145.158:41375] AH00126: Invalid URI in request GET HTTP/1.1 HTTP/1.1
